### DISCUSSION TITLE
Is it possible to colorize branch based on its status?

### DISCUSSION THREAD

**Discussion by ma3yta** [starship/starship#3059] [Q&A] [answered]
I am trying to implement something similar as https://github.com/joelwanner/theme-boxfish
yellow - if the working directory has pending changes
green - if no changes in the working directory
https://github.com/joelwanner/theme-boxfish/blob/master/fish_prompt.fish

![image](https://user-images.githubusercontent.com/660379/133021227-9c6b5b66-6f98-4868-b3b8-23b90fbdb0e1.png)

**Comment by chipbuster** [comment, marked as answer]
It currently is not, but I like the idea! Could you open a feature request issue on this?

Note: I can't promise we'll be able to support it, because this would fundamentally tie the git-state module to the git-branch module, but it would be nice to see if anyone has ideas for how it could be implemented.

**Reply by ma3yta** [reply]
https://github.com/starship/starship/issues/3062
